inductiva/coastal/README.md

```diff
@@ -79,13 +79,13 @@ Now let's perform our simulations with real data!
 Start by loading a bathymetry. This can be done e.g. by reading an
 [ASCII XYZ file](https://emodnet.ec.europa.eu/sites/emodnet.ec.europa.eu/files/public/20171127_DTM_exchange_format_specification_v1.6.pdf),
 a standard format for bathymetry data (for examples, see e.g. the [European Marine Observation and Data Network](https://emodnet.ec.europa.eu/geoviewer/#!/)).
-Here we shall use the bathymetry data from Algarve that you can download [here](/resources/bathymetry/590_HR_Lidar_Algarve.emo)
+Here we shall use the bathymetry data from Algarve that you can download [here](https://downloads.emodnet-bathymetry.eu/high_resolution/590_HR_Lidar_Algarve.emo.zip)
```

Comment:
you're downloading a zip file?

Reply:
Try to follow Hugo tip and use the download from url - see #606 - to obtain the file in the code.

Reply:
Why, and why? (For those two statements). Can you explain further?

Reply:
Luís is helping me with a download_url function that can get and unzip files! That will solve this issue